pkgs/development/tools/devpi-server/default.nix

```diff
@@ -103,6 +104,10 @@ buildPythonApplication rec {
     "devpi_server"
   ];
 
+  postInstall = ''
```

Comment:
consider migrating this to `testVersion`
https://nixos.org/manual/nixpkgs/stable/#tester-testVersion

Reply:
thanks for the hint. i've updated the PR
are passthru tests always run for example when the version is bumped or will this be a manual step (e.g. running `nix-build -A devpi-server.tests` before)

Reply:
ofborg automatically runs passthu tests for all attributes that it picks via commit message ( in this case `devpi-server` )

so for this PR ofborg will run `devpi-server.passthu.tests`, after it finishes eval

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/12160/5447e85e-7cf5-4249-9e04-e958a02d505e)